src/erlcloud_ddb2.erl

```diff
@@ -640,7 +640,9 @@ undynamize_value({<<"M">>, Map}, Opts) ->
 
 -spec undynamize_attr(json_attr(), undynamize_opts()) -> out_attr().
 undynamize_attr({Name, [ValueJson]}, Opts) ->
-    {Name, undynamize_value(ValueJson, Opts)}.
+    {Name, undynamize_value(ValueJson, Opts)};
+undynamize_attr({}, _) ->
+    {}.
```

Comment:
I think this fix is inappropriate.

`jsx` returns `[{}]` for empty objects, and the fix should be translating `[{}]` into `[]`, like we did in:
```erlang
undynamize_object(_, [{}], _) ->
    %% jsx returns [{}] for empty objects
    [];
undynamize_object(PairFun, List, Opts) ->
    [PairFun(I, Opts) || I <- List].
```
thus the following code is correct:
```erlang
undynamize_value({<<"M">>, Map}, Opts) ->
    [undynamize_attr(Attr, Opts) || Attr <- Map].
```
Eventually, we get `[]` as a result for empty map.